tests/network-tests/src/fixtures/proposals/AllProposalsOutcomesFixture.ts

```diff
@@ -75,9 +75,13 @@ export class AllProposalsOutcomesFixture extends BaseFixture {
       }))
       this.debug(
         'Creating proposals:',
-        createProposalsParams.map((p) => p.type)
+        createProposalsParams.map((p) => [p.description, p.details])
+      )
+      const createProposalsFixure = new CreateProposalsFixture(
+        api,
+        query,
+        (createProposalsParams as unknown) as ProposalCreationParams[]
```

Comment:
Type assertion doesn't seem to be necessary here.
In general `as unknown as ...` should always be avoided if possible, it's a very bad practice that can lead to hard-to-debug JS runtime errors

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)